app/src/main/res/values/strings.xml

```diff
@@ -605,6 +605,7 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
 
     <string name="quest_airConditioning_title">"Is this place air-conditioned?"</string>
 
+    <string name="quest_atm_cashin_title">"Does this ATM allows depositing cash?"</string>
```

Comment:
```suggestion
    <string name="quest_atm_cashin_title">"Does this ATM allow depositing cash?"</string>
```

Reply:
Or you could say.

> Can you deposit cash at this ATM?

Reply:
thanks @andrewharvey , added that as it sounds even better!